src/lumigo_tracer/libs/__init__.py

```diff
@@ -1 +1,11 @@
 # TODO - do we need to add to our license something that thanks those repositories?
+
+"""
+Lumigo Tracer tries to use as little external libraries as possible, but sometimes it's just not possible.
+in these cases (When the library is small enough) we copy the code to our repository and use it from there, in order to
+avoid dependency version conflict with the customers dependencies.
+
+The source code is not altered in any way, except for:
+* Imports between libraries are changed to use the local copy.
+* Auto formatting is applied.
```

Comment:
why? we should ignore these paths in black...

Reply:
Correct, ignored them